source/aggregation-tutorials/group-total.txt

```diff
@@ -0,0 +1,225 @@
+.. _node-aggregation-group-total:
+
+===============
+Group and Total
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: code example, node.js, analyze, aggregation
+
+Introduction
+------------
+
+In this tutorial, you can learn how to use the {+driver-short+} to
+group documents and create totals from these groups. The aggregation
+groups documents on a common field and combines information
+from the documents to create new fields.
+
+Aggregation Task Summary
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+This aggregation tutorial demonstrates how to group and analyze data
+describing customer orders. The aggregated result is a list of customers
+who purchased items in 2020. The results include information about
+each customer's order history.
+
+This example uses one collection, ``orders``, which contains documents
+describing individual orders for products in a shop.
+
+An order can only correspond to one customer, so the documents are
+grouped by the ``customer_id`` field, which provides the email address
+of the customer.
+
+Before You Get Started
+----------------------
+
+Before you start following this tutorial, follow the
+instructions in the :ref:`node-agg-tutorial-template-app` section of the
+Aggregation Tutorials page to set up the template app.
+
+Once you set up the app, access the ``orders`` collection by updating
+the collection name placeholder, as shown in the following code:
+
+.. literalinclude:: /includes/aggregation/group-total.js
+   :language: javascript
+   :copyable: true
+   :start-after: start-coll
+   :end-before: end-coll
+   :dedent:
+
+.. note::
+   
+   Since this tutorial accesses documents in only one collection, you
+   can delete any lines in the template app that reference a second
+   collection.
```

Comment:
Suggestion:
I think this admonition should either be part of the main text as an imperative statement (e.g. "delete all lines that contain ...") or omitted.